javascript/packages/orchestrator/src/providers/native/nativeClient.ts

```diff
@@ -431,6 +436,10 @@ export class NativeClient extends Client {
     return false;
   }
 
+  async spawnIntrospector(wsUri: string) {
+    // NOOP
+  }
```

Comment:
```suggestion
  async spawnIntrospector(wsUri: string) {
    // NOOP
    console.log("Not implemented");
  }
```

Reply:
I think is better to show a warning here, with the legend
```
We will not spawn an introspector instance on native provider.
```